document/js-api/index.bs

```diff
@@ -174,7 +176,16 @@ emu-const {
 }
 </style>
 
-This API provides a way to access WebAssembly [[WEBASSEMBLY]] through a bridge to explicitly construct modules from JavaScript [[ECMASCRIPT]].
+<h2 id="intro">Introduction</h2>
+
+By design, the [=scope=] of the WebAssembly [[WEBASSEMBLY]] core specification does not include a description of how WebAssembly programs interact
+with their surrounding execution environment. Instead it defines an abstract [=embedding interface=] between WebAssembly and its environment,
+(called the *embedder*). It is only through this interface that an embedder interacts with the semantics of WebAssembly, and the embedder
+implements the connection between its host environment and the embedding API.
+This document describes the embedding of WebAssembly into JavaScript [[ECMASCRIPT]] environments, including how WebAssembly modules
```

Comment:
I noted that ECMASCRIPT isn't actually listed in the biblio section above. Is that somehow automatic?

Reply:
yes: https://github.com/speced/bikeshed-data/blob/main/data/biblio/biblio-ec.data#L31 (there's probably an easier place to look than these raw files but I'm not sure where it is :)